README.md

```diff
@@ -1,70 +1,266 @@
 # Backendless Chef Stack
 
-Cloudformation Templates to setup our Chef Stack utilizing AWS RDS and Elastic Search as the Chef Backend
+Cloudformation Templates to build out a complete Backendless Chef Configuration
 
-This requires the use of [backendless_chef](https://github.com/HearstAT/cookbook_backendless_chef) cookbook to function.
+**Note:** No longer requires a Cookbook to build.
```

Comment:
I don't see any tags yet.  If you stay under 1.0 (e.g. 0.x.x) then this should likely just be a minor bump.  If you tag the previous version as 1.x then this would likely be a 2.x based on the amount of changes.